README.md

```diff
@@ -46,8 +46,26 @@ me = client.get_me()    # calls /me API endpoint
 print(dumps(me, indent=4))
 ```
 
+For more examples, checkout the [docs](zanshinsdk/docs/README.md).
+
 All operations call `raise_for_status` on the httpx [Response object](https://www.python-httpx.org/api/#response) internally, so any 4xx or 5xx will raise [exceptions](https://www.python-httpx.org/exceptions/).
 
+## Installing
+
+To install the SDK, you can use `pip`. You have two options to install ZanshinSDK:
+- *Essentials*
+  
+Using `pip install zanshinsdk` will install the SDK with all features exception ability to perform onboarding of new Scan Targets. For this, you'll need to install boto3.
```

Comment:
It is possible to create using the create_organization_scan_target function, I believe that using boto3 would be the automated onboarding that includes running the cloudformation